cli/cmd/volume.go

```diff
@@ -74,10 +79,20 @@ var volumeCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: helpVolumeCreateCmd,
 	Run: func(cmd *cobra.Command, args []string) {
-		validateNArgs(cmd, 2, 0)
+		validateNArgs(cmd, 2, 4)
 		volname := cmd.Flags().Args()[0]
 		bricks := cmd.Flags().Args()[1:]
-		fmt.Println("CREATE:", volname, bricks)
+		vol, err := client.VolumeCreate(api.VolCreateReq{
+			Name: volname,
+			Bricks: bricks,
+			Force: flagCreateCmdForce,
+		})
+		if err != nil {
+			log.WithField("volume", volname).Println("volume creation failed")
+			failure(fmt.Sprintf("Volume creation failed with %s", err.Error()), 1)
+		}
+		fmt.Println(vol.ID)
```

Comment:
Add other fields of Volume as summary. (Same format as `gluster volume info` with glusterd1)

Reply:
Added.